### ISSUE TITLE
Ussd response encoding

### ISSUE THREAD

**Issue by bokolob** [pulkin/micropython#82]
Did somebody face problems with the ussd?
cellular.ussd("*100#")  returns me some garbage in the second element. I think GSM7bit_to8bit  should be called depending on response content (a hypothesis).

**Comment by pulkin** [comment]
Yeah that'd be great.

**Comment by ens4dz** [comment]
yes, one of my country operators returns unreadable text from ussd
